config/flags.go

```diff
@@ -384,6 +385,9 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.StringToString(TracingHeadersKey, map[string]string{}, "The headers to provide the trace indexer")
 
 	fs.String(ProcessContextFileKey, defaultProcessContextPath, "The path to write process context to (including PID, API URI, and staking address).")
+
+	// Primary Subnet Config
```

Comment:
what does primary subnet here refer to? is it the primary network or a specific subnet?